static/script/devices/mediaplayer/html5.js

```diff
@@ -577,12 +582,39 @@ require.def(
             },
 
             _enterBufferingSentinel: function() {
-                var notFirstSentinelActivationSinceStateChange = this._sentinelIntervalNumber > 1;
-                if(!this._hasSentinelTimeChanged && !this._nearEndOfMedia && notFirstSentinelActivationSinceStateChange) {
+                var sentinelShouldFire = !this._hasSentinelTimeChangedWithinTolerance && !this._nearEndOfMedia ;
+
+                if (this.getCurrentTime() === 0) {
+                    sentinelShouldFire = this._trustZeroes && sentinelShouldFire;
```

Comment:
Personally I find the mixture of if statements with boolean operators (&&) a little confusing here. For me, this line would be clearer written as:
if (!this._trustZeroes) { sentinelShouldFire = false; }